docs/resources/service_pan.md

```diff
@@ -73,25 +85,28 @@ resource "alkira_service_pan" "test1" {
 - `registration_pin_value` (String) PAN Registration PIN Value.
 - `segment_ids` (List of Number) IDs of segments associated with the service.
 - `size` (String) The size of the service, one of `SMALL`, `MEDIUM`, `LARGE`, `2LARGE`, `4LARGE`, `5LARGE`, `10LARGE`, `20LARGE`.
-- `type` (String) The type of the PAN firewall.
 - `version` (String) The version of the PAN firewall.
 
 ### Optional
 
 - `billing_tag_ids` (List of Number) Billing tag IDs to associate with the service.
 - `bundle` (String) The software image bundle that would be used forPAN instance deployment. This is applicable for licenseType`PAY_AS_YOU_GO` only. If not provided, the default`PAN_VM_300_BUNDLE_2` would be used. However `PAN_VM_300_BUNDLE_2`is legacy bundle and is not supported on AWS. It is recommendedto use `VM_SERIES_BUNDLE_1` and `VM_SERIES_BUNDLE_2` (supports Global Protect).
+- `credential_id` (String) ID of PAN credential.
```

Comment:
I think this one should be in Read Only section? It's computed value and can't be changed by user.